src/main/java/frc/robot/Constants.java

```diff
@@ -39,4 +39,21 @@
     public static final int CONTROL_PANEL_MOTOR = 15;
     public static final int CONTROL_PANEL_SOLENOID_FWD = 3;
     public static final int CONTROL_PANEL_SOLENOID_REV = 1;
+
+    // RobotContainer object configuration
+    public static final double DRIVE_GEARING = 7.56;
+    public static final double DRIVE_WHEEL_DIAMETER = 6.0;
```

Comment:
Let's specify units for this and other constants.

Reply:
I just noticed this must be converted to meters. A good reason to specify units.